### DISCUSSION TITLE
How to provide null values to stmt.Exec?

### DISCUSSION THREAD

**Discussion by renatospaka** [sijms/go-ora#424] [Q&A] [answered]
I have a huge table from a legacy system and I need to insert values in it.

However, there are several nullable fields and so far I didn't figure out how to provide `null`  values instead of `zeros` or `empty strings` to `stmt.ExecContext` command. They need to be `null`

The `stmt` can refer to an **Insert**, **Upsert** or **Update**.

Does anyone know how?

**Comment by sijms** [comment, marked as answer]
use sql.NullString data type

**Reply by renatospaka** [reply]
Hey @sijms , thank you.

BTW, can I integrate a logger with `go-ora`? If so, is there an exemple I can read?

**Comment by sijms** [comment]
I update documentation in readme file so it is more organized and understandable
please check you will find a section for logging
